index.js

```diff
@@ -27,6 +27,9 @@ module.exports = {
       .for('ember-qunit')
       .lt('5.0.0-beta.1');
```

Comment:
Should we also update this to point to the full major?

Reply:
The main reason we used the beta was because this was the first version that used the new setup, so I think `5.0.0-beta.1` is still the correct value.

Reply:
Sure, it just seems odd to have beta be the value we test for, though technically it's correct.